src/utils/_mixins.scss

```diff
@@ -0,0 +1,15 @@
+@mixin grid($columns, $screen-size, $card-size,) {
+  @media (min-width: $screen-size) {
+    grid-template-columns: repeat($columns, $card-size);
+  }
+}
```

Comment:
Mixin `grid` should include `display: grid;` This mixin worsens code readability. My suggestion is to remove it and use media queries inside the class.

To simplify writing media add mixins like that

```
@mixin onTablet {
  @media (max-width: 768px) {
    @content
  }
}
```

Reply:
all suggestions were implemented